kafka-ui-react-app/src/components/common/NewTable/SelectRowHeader.tsx

```diff
@@ -6,7 +6,7 @@ const SelectRowHeader: React.FC<HeaderContext<unknown, unknown>> = ({
   table,
 }) => (
   <IndeterminateCheckbox
-    checked={table.getIsAllPageRowsSelected()}
+    checked={table.getIsAllRowsSelected()}
     indeterminate={table.getIsSomePageRowsSelected()}
     onChange={table.getToggleAllPageRowsSelectedHandler()}
```

Comment:
`getToggleAllRowsSelectedHandler()` since they all should rely on on Rows selected instead of a PageRows